aws/src/main/java/org/apache/iceberg/aws/glue/GlueCatalog.java

```diff
@@ -162,6 +189,18 @@ private String cleanWarehousePath(String path) {
 
   @Override
   protected TableOperations newTableOps(TableIdentifier tableIdentifier) {
+    if (catalogProperties != null) {
```

Comment:
Should this be logically `if (lakeFormationEnabled)`?

Reply:
The intention is to allow other table specific features in the future, for example if we want to introduce table-specific s3 tags.